src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -392,43 +392,6 @@ public static String getPlayerTeleporter(String playerName) {
     //TODO: REMOVE THIS STATIC CRAP - END
 
 
-    // For testing purposes only //
-
-    private File serverFolder = new File(System.getProperty("user.dir"));
-
-    /**
-     * This is for unit testing.
-     *
-     * @param loader The PluginLoader to use.
-     * @param description The Description file to use.
-     * @param dataFolder The folder that other datafiles can be found in.
-     * @param file The location of the plugin.
-     */
-    public MultiverseCore(JavaPluginLoader loader, PluginDescriptionFile description, File dataFolder, File file) {
-        super(loader, description, dataFolder, file);
-    }
```

Comment:
This is needed for unit testing. Not sure how the status checks are passing on this branch...